### DISCUSSION TITLE
Type String Issue

### DISCUSSION THREAD

**Discussion by GraceOb98** [community/community#33124] [Other Feature Feedback, Questions, & Ideas]
Hello, 
For developing a [video editor app](https://modstoapk.com/blue-kinemaster-pro-mod-apk/), the problem I faced is; that the "type String cannot be converted into JSON object"
How can I solve this?
![305405512_8685417718142380_1753761419366916907_n](https://user-images.githubusercontent.com/111198780/190317810-70eb270b-fa05-4030-aaa5-d8ab91b18232.jpg)
Thanks in Advance!

**Comment by jfullstackdev** [comment]
maybe this thread will help too

https://stackoverflow.com/questions/10267910/jsonexception-value-of-type-java-lang-string-cannot-be-converted-to-jsonobject
